common-lib/src/cmr/common/util.clj

```diff
@@ -469,6 +469,17 @@
     (catch NumberFormatException _
       false)))
 
+;; Differences from numeric-string? above:
+;;   More strict - For example, returns false on string "0D".
+;;   Works on any sequence of characters, not just strings.
+(defn numeric? [s]
```

Comment:
Would prefer styleguide recommendation and move this to a docstring and reference `numeric-string?` from within there. See https://guide.clojure.style/#document-references

Reply:
Yes that looks much better. done.